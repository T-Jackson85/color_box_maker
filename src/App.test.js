import React from "react";
import { render, screen } from '@testing-library/react';
import App from './App';

it('renders App (smoke test)', () => {
  render(<App />);
});

it("snapshot test of App", function() {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
