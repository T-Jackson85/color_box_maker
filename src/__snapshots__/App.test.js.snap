// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test of App 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <form>
      <div>
        <label
          class="Form-Label"
          for="height"
        >
          Height: 
        </label>
        <input
          class="Form-Input"
          id="height"
          name="height"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          class="Form-Label"
          for="width"
        >
          Width: 
        </label>
        <input
          class="Form-Input"
          id="width"
          name="width"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          class="Form-Label"
          for="backgroundColor"
        >
          Background Color: 
        </label>
        <input
          class="Form-Input"
          id="backgroundColor"
          name="backgroundColor"
          type="text"
          value=""
        />
      </div>
      <button
        class="Form-Button"
        id="formButton"
      >
        Add a box
      </button>
    </form>
  </div>
</DocumentFragment>
`;
